game/scripts/npc/abilities/doom_bringer_devour.txt

```diff
@@ -45,7 +45,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "regen"                                           "10 20 30 40 70 100" //OAA
+        "regen"                                           "8 16 24 32 70 100" //OAA
```

Comment:
I would probably nerf the last 2 levels as well. Something like 50/80 instead of 70/100

Reply:
OAA levels are fine.